Library/Homebrew/cmd/desc.rb

```diff
@@ -17,7 +17,7 @@ module Homebrew
   def desc_args
     Homebrew::CLI::Parser.new do
       usage_banner <<~EOS
-        `desc` [<options>] (<text>|`/`<text>`/`|<formula>)
+        `desc` [<options>] <text>|`/`<text>`/`|<formula> [<text>|`/`<text>`/`|<formula> ...]
```

Comment:
FYI this new usage banner is a little too long:

```console
$ brew help desc
Usage: brew desc [options] text|/text/|formula [text|/text/|formula
...]

Display formula's name and one-line description. Formula descriptions are
cached; the cache is created on the first search, making that search slower than
subsequent ones.

  -s, --search                     Search both names and descriptions for
                                   text. If text is flanked by slashes, it
                                   is interpreted as a regular expression.
  -n, --name                       Search just names for text. If text is
                                   flanked by slashes, it is interpreted as a
                                   regular expression.
  -d, --description                Search just descriptions for text. If
                                   text is flanked by slashes, it is
                                   interpreted as a regular expression.
      --debug                      Display any debugging information.
  -q, --quiet                      Make some output more quiet.
  -v, --verbose                    Make some output more verbose.
  -h, --help                       Show this message.
```

We'll probably want to shorten it in the future so it fits on one line. Also probably worth having a rubocop to check for this (if it's not automatically generated)